### PR TITLE
[knx] Add list of supported DPTs

### PR DESCRIPTION
**Pull request by holgerfriedrich**
Added a table containing all supported DPTs.

Any suggestions how to improve the table view in md?

### CODE REVIEW

**Review by stefan-hoehn — APPROVED** [review]
Thanks for the comprehensive extension of the documentation and even more even completely flawless 👏🏻

**Review by stefan-hoehn — APPROVED** [review]

File: bundles/org.openhab.binding.knx/README.md [hunk 1]
```diff
@@ -292,22 +444,22 @@ Additional DPTs have been introduced to add functionality:
...
+- All information can be looked up in ETS and provided separately: `tunnelDeviceAuthentication`, `tunnelUserPassword`. `tunnelUserId` is a number that is not directly visible in ETS, but can be looked up in keyring export or deduced (typically 2 for the first tunnel of a device, 3 for the second one, ...). `tunnelUserPasswort` is set in ETS in the properties of the tunnel (below the IP interface, you will see the different tunnels listed) and denoted as "Password". `tunnelDeviceAuthentication` is set in the properties of the IP interface itself; check for the tab "IP" and the description "Authentication Code".
```
Comment (lsiepel) [inline bundles/org.openhab.binding.knx/README.md:457]:
```suggestion
- All information can be looked up in ETS and provided separately: `tunnelDeviceAuthentication`, `tunnelUserPassword`.
`tunnelUserId` is a number that is not directly visible in ETS, but can be looked up in keyring export or deduced (typically 2 for the first tunnel of a device, 3 for the second one, ...).
`tunnelUserPasswort` is set in ETS in the properties of the tunnel (below the IP interface, you will see the different tunnels listed) and denoted as "Password".
`tunnelDeviceAuthentication` is set in the properties of the IP interface itself; check for the tab "IP" and the description "Authentication Code".
```

File: bundles/org.openhab.binding.knx/README.md [hunk 2]
```diff
@@ -126,10 +127,10 @@ The `hsb` address supports DPT 232.600 (RGB), 242.600 (xyY), and 251.600 (RGBW).
...
 This is supported as "vendor-specific DPT" with a value of 232.60000.
 
-RGBW (DPT 251.600) can either be converted to HSBType, or be represented two items: a HSBType for RGB and an additional PercentType for W channel.
+RGBW (DPT 251.600) can either be converted to HSBType, or represented by two items: a HSBType for RGB and an additional PercentType for the W channel.
 Default handling for RGBW is to use separate items.
-Note that this also requires two frames being sent out separately when these elements are sent to the bus, as the binary representation uses a partially populated KNX frame.  
-Alternatively, a single HSB item can be used. Conversion to a single HSBType will loose the exact setting for W, and will reconstruct it when a conversion to RGBW is required.
+Note that this also requires two frames to be sent out separately when these elements are sent to the bus, as the binary representation uses a partially populated KNX frame.  
+Alternatively, a single HSB item can be used. Conversion to a single HSBType will lose the exact setting for W, and will reconstruct it when a conversion to RGBW is required.
```
Comment (lsiepel) [inline bundles/org.openhab.binding.knx/README.md:133]:
```suggestion
Alternatively, a single HSB item can be used.
Conversion to a single HSBType will lose the exact setting for W, and will reconstruct it when a conversion to RGBW is required.
```

**Review by lsiepel — CHANGES_REQUESTED** [review]
Very minor comments.

File: bundles/org.openhab.binding.knx/README.md [hunk 3]
```diff
@@ -273,6 +274,156 @@ With `*-control` channels, the state is not owned by any device on the KNX bus,
...
 
+## Mapping DPTs to openHAB Types
+
+Datapoint Types (DPTs) define how the content of a KNX telegram is interpreted.
+
+The following table is a complete list of DPTs currently supported by openHAB.
+OpenHAB supports all DPTs supported by the corresponding release of Calimero library, plus a few [specific additions](#special-dpts).
+
+The default mapping is given, however DPTs can be overwritten.
+KNX frames do not contain information about the encoding or DPT, so and DPT with a compatible byte size and a useful encoding can be used. A good example for this are bitfields represented as String which are mapped to a DPT of DecimalType for handling in rules.
```
[the remark this answers is not in the document]
Comment (holgerfriedrich) [reply bundles/org.openhab.binding.knx/README.md]: this has already been resolved in my second commit

File: bundles/org.openhab.binding.knx/README.md (hunk 2 above)
Comment (holgerfriedrich) [reply bundles/org.openhab.binding.knx/README.md]: Resolved. I did not know how to do it without breaking the enumeration. Adding a space as first character of the new line seems to work.

**Review by lsiepel — APPROVED** [review]
LGTM

**Pull request merged**